arangod/Aql/DocumentProducingHelper.cpp

```diff
@@ -680,7 +680,8 @@ IndexIterator::CoveringCallback aql::getCallback(
         };
         context.getPhysical().lookup(
             context.getTrxPtr(), token, cb,
-            {.readOwnWrites = static_cast<bool>(context.getReadOwnWrites())});
+            {.readOwnWrites = static_cast<bool>(context.getReadOwnWrites()),
+             .countBytes = true});
```

Comment:
The description says: 
```These metrics are not exposed by default.```

Here it is set to be statically true. Is this correct? Maybe I'll understand as soon as I am finished with the review.

Reply:
Yes, this is correct.
Only certain operations will increase the metrics. All of the operations that do have the `countBytes` attribute set to true. All others have it set to `false`.

The counting will only happen, regardless of the value of this boolean, if certain other conditions are fulfilled, e.g. a certain startup option is set.